js/app.js

```diff
@@ -79,9 +77,36 @@ function render (Store) {
 
 };
 
-
 //salmon image
 var img = document.createElement('img');
 img.setAttribute('src', 'images/salmon.png');
 img.setAttribute('alt', 'cute picture of a salmon');
 article.appendChild(img);
+
+//for verifying whether store name is already in use in the table
+var storeNames = ['alki', 'first and pike', 'seatac airport', 'capital hill', 'seattle center'];
+
+var cookieCruncher = document.getElementById('addCookieCruncher');
+// event listeners ened to know: what event do they care about, and what do they want to do when it happens.
+cookieCruncher.addEventListener('submit',
+function (event) {
+  event.preventDefault();
+  var name = event.target.name.value;
+  var cookieAverage = event.target.cookieAverage.value;
+  var hoursOpen = ['','6am','7am','8am', '9am', '10am','11am', '12pm', '1pm','2pm', '3pm', '4pm','5pm', '6pm', '7pm','8pm'];
+  var minCookie = event.target.minCookie.value;
```

Comment:
I would use typeof for minCookie, maxCookie, and cookieAverage. Do you get number? If not, what can we use to convert it to numbers?